app/models/user.rb

```diff
@@ -37,6 +37,14 @@ class User < ActiveRecord::Base
   has_attached_file :image, styles: { thumb: "50x50>" }
   validates_attachment_content_type :image, content_type: /\Aimage\/.*\z/
 
+  def self.count_of_members_and_supervisors?(current_user, member)
+    current_supervisors_count = User.where(role: ROLE[:supervisor]).count
```

Comment:
User.where(role: ROLE[:supervisor]).count -> this will give all the users for all companies 
User.where(role: ROLE[:supervisor]).count -> @current_company.users.where...